core/shared/src/test/scala/zio/prelude/CovariantSpec.scala

```diff
@@ -18,7 +23,8 @@ object CovariantSpec extends DefaultRunnableSpec {
         testM("cause")(checkAllLaws(Covariant)(GenFs.cause, Gen.anyInt)),
         testM("chunk")(checkAllLaws(Covariant)(GenF.chunk, Gen.anyInt)),
         testM("exit")(checkAllLaws(Covariant)(GenFs.exit(Gen.causes(Gen.anyInt, Gen.throwable)), Gen.anyInt)),
-        testM("nonEmptyChunk")(checkAllLaws(Covariant)(GenFs.nonEmptyChunk, Gen.anyInt))
+        testM("nonEmptyChunk")(checkAllLaws(Covariant)(GenFs.nonEmptyChunk, Gen.anyInt)),
+        testM("chunk . option")(checkAllLaws(Covariant)(chunkOptionGenF, Gen.anyInt))
```

Comment:
This is already tested in `TraversableSpec`, so it doesn't make sense to test it again here, no?

Reply:
Well, it's a different piece of code. But the implementation is obviously the same.

Reply:
`TraversableSpec` literally tests (also) the Covariant laws, so we shouldn't need to test them again here. Or am I missing something? What do you mean by
> different piece of code

:thinking:

Reply:
It's a different `compose` method

Reply:
Oh, of course! :facepalm:  Thanks for the explanation :smile_cat: